guide/10-mapping-and-visualization/usage-of-visualVariables-and-other-renderer-options-in-map-widget.ipynb

```diff
@@ -12,7 +12,7 @@
    "cell_type": "markdown",
```

Comment:
<p>@CMPeng I'm not sure if I'm missing something, but it seems that we're only using the id from the `map6_layers`. Do we need such a big dict with popup info, field info, etc?</p><p><br></p>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/410/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>